app/views/groups/show.html.erb

```diff
@@ -52,14 +58,15 @@
       <%= link_to t("groups.show.assignment.add_assignment_btn"), new_group_assignment_path(@group), class: "btn primary-button groups-primary-button" %>
     <% end %>
   </div>
-  <hr>
+    <% if @group.assignments.empty? %>
+      <hr>
+      <% if @group.assignments.empty? %>
+        <span class="null-text"><%= t("groups.show.no_assignments_html") %></span>
+      <% end %>
+    <% end %>
   <table class="table table-borderless groups-table">
     <thead class="groups-table-heading">
       <tr>
-        <th><%= t("groups.show.assignment.assignment_table_head.name") %></th>
-        <th><%= t("groups.show.assignment.assignment_table_head.deadline") %></th>
-        <th><%= t("groups.show.assignment.assignment_table_head.grading") %></th>
-        <th><%= t("groups.show.assignment.assignment_table_head.actions") %></th>
         <% unless policy(@group).admin_access? %>
           <th><%= t("groups.show.assignment.assignment_table_head.grade") %></th>
         <% end %>
```

Comment:
Why remove this?
```
  <hr>
  <table class="table table-borderless groups-table">
    <% if @group.assignments.empty? %>
      <span class="null-text"><%= t("groups.show.no_assignments_html") %></span>
    <% else %>
      <thead class="groups-table-heading">
        <tr>
          <th><%= t("groups.show.assignment.assignment_table_head.name") %></th>
          <th><%= t("groups.show.assignment.assignment_table_head.deadline") %></th>
          <th><%= t("groups.show.assignment.assignment_table_head.grading") %></th>
          <th><%= t("groups.show.assignment.assignment_table_head.actions") %></th>
          <% unless policy(@group).admin_access? %>
            <th><%= t("groups.show.assignment.assignment_table_head.grade") %></th>
          <% end %>
        </tr>
      </thead>
    <% end %>
```
Try this.
What we want is if there are no assignment render no_assignment_message else render table head and body!

Reply:
Oh sorry I misread it and acted in a hurry. Correcting it 👍

Reply:
![Screenshot 2022-12-08 at 7 02 25 PM](https://user-images.githubusercontent.com/85568177/206459279-a06a2063-92a5-4843-ad54-c0fa826fb487.png)
Looks like this now